Mobile nav styles

danelegassick opened the pull request:
Updated nav to have hamburger menu and microanimation to turn into x icon.
Added back arrow icon and logo
Styled off screen nav + slide in
Added div for shadow behind the slide-in nav that has opacity shift on click of hamburger menu

*2 things*
1. this back arrow is not functional, noting this so you can take care of this when you get around to it.
2. this has opened my eyes to problems with homepage content sizing - will need to open a new pr to fix change from:
width: X rem; -> width: 100%; max-width: X rem

danelegassick committed:
Initial Commit (Base functionality)

FINALLY

minor fix on how the translate button is centered.

created base functionality for hamburger styling. menu slides out from right. no styling applied to menu.

need to add in backshadow, other nav items (back arrow and logo), styling to offscreen slide-in menu

this took too long.

danelegassick committed:
logo and back arrow

Added and placed logo and back arrow for top nav

danelegassick committed:
Off Screen Menu Styling

Added logo and hrs to break apart links.

Added background shadow behind menu on open.

Need to re-adjust how width of dropdowns is calculated - right now creates problem with small screens and overflow. This will be seperate PR, but noting here.

These

alexisraya commented:
@danelegassick this looks great. I just have one question regarding the background for the nav? Why is it transparent?

danelegassick committed:
Fixed header transparency

Header now has background color that is sticky to top of page.

Fixed issue with shadow div blocking interaction with elements on page

(this got read of the opacity fade on closing of hamburger menu, but the trade off is that you actually can use the app!)

alexisraya commented:
Thanks for the fix! Merging now
